test/humanizer.js

```diff
@@ -99,7 +99,7 @@ describe('humanizer', function () {
     var h = humanizer({ round: true })
 
     assert.equal(h(3692131200000, { largest: 1 }), '117 years')
-    assert.equal(h(3692131200000, { largest: 2 }), '117 years')
+    assert.equal(h(3692131200000, { largest: 2 }), '116 years, 12 months')
```

Comment:
Yes, I change it because:
**3692131200001** ms are exactly **116 years, 11 months, 4 weeks, 1 day, 4 hours, 30 minutes**.
So if you want to show only the two largest units and round them you have to show **116 years, 12 months**.